### DISCUSSION TITLE
Using Docker-lidarr-extended only for Music VIdeos

### DISCUSSION THREAD

**Discussion by chrisjameschamp** [RandomNinjaAtk/docker-lidarr-extended#207] [Q&A] [answered]
How would I use this to only get music videos, I don't want to download or monitor any actual music tracks, I just want it to manually import specific songs that I would like and for it to only get the music video for it.  Is this possible?

**Comment by RandomNinjaAtk** [comment, marked as answer]
Music Video only is simple, just only enable the video script and don't add any indexers/clients to lidarr...

As far as singles goes, lidarr is designed around albums, so unless its a singles album, you can't just grab random tracks...

**Reply by chrisjameschamp** [reply]
Alright I will give that a try

**Reply by KINGLIFER** [reply]
How did it go?
